test/util_test.py

```diff
@@ -1,7 +1,11 @@
+# fmt: off
+# pylint: disable=W0703
```

Comment:
why are you turning black and pylint off?

Reply:
🙈